amundsen_application/static/js/ducks/notification/api/v0.ts

```diff
@@ -2,10 +2,15 @@ import axios from 'axios';
 import { NotificationType, SendNotificationOptions } from 'interfaces'
 
 export function sendNotification(recipients: Array<string>, sender: string, notificationType: NotificationType, options?: SendNotificationOptions) {
-  return axios.post(`/api/mail/v0/notification`, {
-    recipients,
-    sender,
-    notificationType,
-    options,
-  })
+  return axios({
+    data: {
+      recipients,
+      sender,
+      notificationType,
+      options,
+    },
+    method: 'post',
+    url: `/api/mail/v0/notification`,
+    timeout: 5000
```

Comment:
Let's remove `timeout: 5000` before merge. It's not needed and could cause problems if the mail client suffers some latency issue and happens to take more than 5 seconds to compete the request.